frontend/webapp/views/main/templates/hijack.htm

```diff
@@ -704,14 +746,19 @@ <h1 class="page-header">Viewing Hijack <small id="hijack_status"></small></h1>
 
                 $( "#seenHijackWithdraw_text").html( peers_html.join('') );
 
-                if(data['resolved'] == true || data['ignored'] == true){
+                if(hijack_obj['resolved'] == true || hijack_obj['ignored'] == true){
                     $("#resolved_button").prop("disabled", true);
                     $("#mitigation_button").prop("disabled", true);
                     $("#ignored_button").prop("disabled", true);
-                }else if(data['under_mitigation'] == true || data['withdrawn'] == true || data['outdated'] == true){
+                }else if(hijack_obj['under_mitigation'] == true || hijack_obj['withdrawn'] == true || hijack_obj['outdated'] == true){
```

Comment:
if under_mitigation is enabled then also the ignore/resolve buttons should be activated (remove disable I mean)

Reply:
Done